cluster/addons/cluster-monitoring/google/heapster-controller.yaml

```diff
@@ -1,12 +1,16 @@
 {% set base_metrics_memory = "200Mi" -%}
 {% set metrics_memory = base_metrics_memory -%}
+{% set base_metrics_cpu = "80m" -%}
+{% set metrics_cpu = base_metrics_cpu -%}
 {% set base_eventer_memory = "200Mi" -%}
 {% set eventer_memory = base_eventer_memory -%}
 {% set metrics_memory_per_node = 4 -%}
+{% set metrics_cpu_per_node = 1 -%}
 {% set eventer_memory_per_node = 500 -%}
 {% set num_nodes = pillar.get('num_nodes', -1) -%}
 {% if num_nodes >= 0 -%}
   {% set metrics_memory = (200 + num_nodes * metrics_memory_per_node)|string + "Mi" -%}
+  {% set metrics_cpu = (num_nodes + 80)|string + "m" -%}
```

Comment:
OK - I think I know how to solve it relatively easy. We should add another flag to pod-nany that will allow us to add a given number of millicores per given number of nodes. If we set the second flag to 2, and the existing flag to 1millicore, this will result in adding 1 millicore per 2 nodes. This should be easy to do - I can help with that tomorrow.